src/OrchardCore.Modules/OrchardCore.Autoroute/Razor/AutoroutePartRazorHelperExtensions.cs

```diff
@@ -5,7 +5,9 @@
 using OrchardCore.ContentManagement;
 using OrchardCore.ContentManagement.Routing;
 
+#pragma warning disable CA1050 // Declare types in namespaces
```

Comment:
All razor helper extensions in the whole solution are not defined under a namespace, so that I think they are all suggested by intellisense in razor views without having to add `using` declarations.

So I kept them as is to not break existing razor views using them